crates/polars-expr/src/expressions/slice.rs

```diff
@@ -110,6 +110,11 @@ impl PhysicalExpr for SliceExpr {
                 .collect::<PolarsResult<Vec<_>>>()
         })?;
         let mut ac = results.pop().unwrap();
+
+        if ac.is_aggregated() {
+            polars_bail!(InvalidOperation: "cannot slice() an aggregated value")
```

Comment:
drive-by - I noticed we currently silently ignore `slice()` after aggregations - I've made this raise instead.